### ISSUE TITLE
Osu showing no video output

### ISSUE THREAD

**Issue by imrolii** [ppy/osu-stable-issues#730]
When I launch Osu on my laptop, the music begins to play and the voiceover starts to talk, but I cannot see anything. When I move my mouse and click it is interacting with the game as the audio changes but I still can't see anything.

I've tried the 'repair osu!' shortcut to see if that would help but it hasn't, and neither did pressing alt+enter to go to windowed-mode.

Laptop Specs:
AMD A9-9425
4GB RAM
1TB HDD
AMD Radeon R5 Graphics (512MB RAM - Integrated)
Windows 10 Home x64 - 20H2

Any help would be greatly appriciated since it was working beforehand, thanks ever so much :-)

**Comment by peppy** [comment]
Can you try switching to cutting edge via the repair screen?

**Comment by imrolii** [comment]
Thanks for your response,

Issue is now fixed after a full system reboot. Probably should have tried that first :p

Farewell!
